common/system/filtering/filters.cpp

```diff
@@ -387,15 +387,21 @@ static void filterPositionZ(SensorData_s* curSensorVals, bool hasPassedApogee) {
   // TODO check what order these "should" run in. Worst case we're off by one
   // iteration We don't update accelerations till after this, so the z
   // acceleration should still be from the last timestep
+
+  // Updates Kalman filter timestamps to the current loop and uses new values to
+  // calculate the current dt
+  kalman.pushTimeStamps(curSensorVals->timestampMs);
```

Comment:
Is it possible for you to combine these two function calls into one function call, at least for external users of the Kalman filter? It's ok if they're still separate functions internally but it would be good to only have to call one function to do this. Maybe `updateDt()` or something